eng/pipelines/optprof.yml

```diff
@@ -170,6 +170,6 @@ stages:
     - template: \steps\powershell\execute-script.yml@DartLabTemplates
       parameters:
         displayName: Create OptimizationInputs Tag in Repository
-        condition: or(and(succeeded(), in(variables['Build.Reason'], 'ResourceTrigger')), eq('${{parameters.optEnableOptProfTagging}}', 'true'))
+        condition: and(succeeded(), or(in(variables['Build.Reason'], 'ResourceTrigger'), eq('${{parameters.CreateGitTagOnManualRun}}', 'true')))
```

Comment:
I imagine the tags are happening in the internal repo?

Reply:
yes, that's right. There's no benefit to making these tags available on github. THey're an implementation detail about how OptProf works, which we only use/need for official builds, which also happen on the internal repo.